### PR TITLE
[JS] Run the named actions before running the format when the file is open (issue #15818)

### LINKED ISSUE #15818: Can't edit some fields in the DB (Deutsche Bahn) reimbursment form
Attach (recommended) or Link to PDF file here:

[fahrgastrechteformular.pdf](https://github.com/mozilla/pdf.js/files/10216567/fahrgastrechteformular.pdf)

https://assets.static-bahn.de/dam/jcr:38fc20e5-9179-4abb-94c5-ebddab941b33/fahrgastrechteformular.pdf

Configuration:
- Web browser and its version: Firefox Nightly 109.0a1 (2022-12-09)
- Operating system and its version: Linux
- PDF.js version: :shrug: 
- Is a browser extension: it's Firefox :shrug:

Steps to reproduce the problem:
1. open attached PDF
2. try to fill in the fields, especially the ones left empty in the screenshot below

What is the expected behavior?

image

What went wrong?

image

### PR DESCRIPTION
**Pull request by calixteman**
It's a follow-up of #14950: some format actions are ran when the document is open but we must be sure we've everything ready for that, hence we have to run some named actions before runnig the global format.
In playing with the form, I discovered that the blur event wasn't triggered when JS called `setFocus` (because in such a case the mouse was never down). So I removed the mouseState thing to just use the correct commitKey when blur is triggered by a TAB key.

### CODE REVIEW

File: src/display/annotation_layer.js [hunk 1]
```diff
@@ -2611,7 +2613,6 @@ class AnnotationLayer {
         enableScripting: parameters.enableScripting,
         hasJSActions: parameters.hasJSActions,
         fieldObjects: parameters.fieldObjects,
-        mouseState: parameters.mouseState || { isDown: false },
```
Comment (Snuffleupagus) [inline src/display/annotation_layer.js:2614]: I'm not at all sad to see that parameter go!

However there's still some `mouseState` occurrences found in this file, in both code and JSDocs, so please go through and remove those as well.

**Review by Snuffleupagus — APPROVED** [review]
r=me, with the comment addressed; thank you!

File: src/display/annotation_layer.js (hunk 1 above)
Comment (calixteman) [reply src/display/annotation_layer.js:2614]: You know what ??.... me neither ;)

**Pull request merged**